podman-image-daily/unregister-handler.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Unregister x86_64 handler if necessary
+After=rosetta-activation.service
+ConditionPathExists=/mnt/rosetta
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/bin/sh -c "echo -1 > /proc/sys/fs/binfmt_misc/qemu-x86_64"
```

Comment:
im going to merge this but i want to point out that in your podman pr for ignition, you will need to make this specific to apple silicon only.